### PR TITLE
Initial changes to the deegree documentation for OSGeoLive15

### PR DESCRIPTION
**Pull request by julianzz98**
since I didnt build the docs locally, the changes NEED to be tested.

### CODE REVIEW

File: doc/overview/deegree_overview.rst [hunk 1]
```diff
@@ -122,7 +126,4 @@ Details
 
 
 @VMDK_deegree@
-@QUICKSTART_deegree@
-
-.. presentation-note
```
Comment (cvvergara) [inline doc/overview/deegree_overview.rst:127]: Looks like the note for the [presentation](https://osgeo.github.io/OSGeoLive-doc/en/presentation.html#/7/1) was deleted: (Press "s" and the notes will pop out in a different window)

Comment (cvvergara) [reply doc/overview/deegree_overview.rst:127]: image

File: doc/quickstart/deegree_quickstart.rst [hunk 2]
```diff
@@ -45,46 +46,36 @@ Load a workspace
...
-
 What next?
 ==========
 
 This was just a short introduction to the features of deegree 3. 
 There is more information in the demo installations and on the deegree website.
 
-  * deegree online demo - Available here: https://www.deegree.org/demo/
-
   * deegree website - Available here: https://www.deegree.org
```
Comment (cvvergara) [inline doc/quickstart/deegree_quickstart.rst:77]: The syntax for bullets:
The bullet is at the first column of the line.
There are no blank lines in between when the bullet is at the same level.

```
* b1
* b2
* b3

   * b3.1
   * b3.2

* b4
```

**Review by cvvergara — CHANGES_REQUESTED** [review]
Hi @julianzz98 
I am in charge of the technical checks.
There are some changes to be done.

**Comment by cvvergara** [comment]
TODO I will merge and fix the issues.

**Comment by julianzz98** [comment]
Hello and sorry for not responding, I am on my way to Brazil right now and can look into fixing it only next week. If you can make the changes, I would appreciate it! Greetings

**Pull request merged**